src/content/docs/tutorial-create-alerts/create-nrql-alerts.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: "Create a NRQL alert"
+metaDescription: "How to create a NRQL alert with New Relic"
+---
+
+import alertsNRQL from 'images/alerts_screenshot-crop_NRQL-query.webp'
+
+import alertsNewCreate from 'images/alerts_screenshot-crop_create-new-query.webp'
+
+While using charts is a great way to create alert conditions from premade queries, sometimes you need to create alert conditions from scratch. The process for creating a new alert from a NRQL query is almost the exact same as creating it from a chart: the key difference is that you write the NRQL query manually instead of using a pre-generated one. 
+
+<img
+    title="NRQL alert"
+    alt="A screenshot displaying an alert condition created with a simple NRQL query"
+    src={alertsNRQL}
+/>
+
+<Callout variant="tip">
+Creating alert conditions from NRQL queries requires a basic understand of NRQL, New Relic's Query Language. See our [NRQL docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn more about NRQL, including [how to use it](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and an [introductory tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
```

Comment:
can we reframe this to be more positive? e.g.

> NRQL stands for New Relic's Query Language—a standardized query method that powers all our charts, and which you can use to analyze and query your data. If this is your first time working with NRQL, you may want to check out  our NRQL docs to learn more about NRQL, including how it works and an introductory tutorial.

Reply:
Can and done!